### DISCUSSION TITLE
Email on dracula's website is broken

### DISCUSSION THREAD

**Discussion by pranavdhamanage** [dracula/dracula-theme#867] [Q&A] [answered]
Hi,
Need to connect with customer service or someone regarding some questions on theme. Please can anyone share an email id to which I can reach out to

**Comment by luxonauta** [comment, marked as answer]
Hi 👋🏻 Pranav, how are you?

You can contact us through the [**support email**](mailto:support@draculatheme.com); however, if the theme in question is an **open-source project theme**, please **open an Issue** in the same repository here in our GitHub organization.
